testing/integration/utils.py

```
@@ -164,65 +304,93 @@ def start_valkey_cluster(
+                valkey_cli_args.append(f"127.0.0.1:{port}")
+            valkey_cli_args.extend(["--cluster-replicas", str(replica_count)])
+            if password:
+                valkey_cli_args.extend(["-a", password])
+
+            logging.info("Creating valkey cluster with command: %s", valkey_cli_args)
+
+            timeout = 60
+            now = time.time()
+            while time.time() - now < timeout:
+                try:
+                    subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)